### PR TITLE
fixed:New page html

### PR DESCRIPTION
**Pull request by User-Business**

### CODE REVIEW

**Review by cadegore — CHANGES_REQUESTED** [review]
@User-Business Please update `style.css` to `styles.css`

File: Art/User-Business-portfolio/index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Meu Portfólio</title>
+  <link rel="stylesheet" href="style.css">
```
Comment (LaurelineP) [inline Art/User-Business-portfolio/index.html:7]: # Incorrect css file name
Currently the filename is matching the link's href
However this filename is incorrect according to both README requirements and the checklist.

- [ ] Please change your file name accordingly for both the css file name and this href

File: Art/User-Business-portfolio/style.css
Comment (LaurelineP) [inline Art/User-Business-portfolio/style.css:1]: # Incorrect css file name
Currently the filename is matching the link's href
However this filename is incorrect according to both README requirements and the checklist.

- [ ] Please change your file name accordingly for both the css file name and this href

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @User-Business 
Thanks for your contribution
Please address the provided  feedbacks in order for us to validate your PR for a merge request

Also:
your code is indeed html and css but does not seem to do any animations.
The end challenge is to create an animation - most likely this should not be accepted as contribution.
This is why there is currently an invalid tag.
If this is the case, I'll check with maintainers, but this PR could be closed and this will be for the mentioned reasons.

Meanwhile, you could address the previous feedbacks :)

**Pull request closed without merging**